### DISCUSSION TITLE
Can I configure syntax highlighting?

### DISCUSSION THREAD

**Discussion by Lohoyo** [codesandbox/codesandbox-client#6095] [Q&A] [answered]
Hello!

I have a template that has a `.san` file.
`.san` files are like `.vue` files.
However, `.vue` files have syntax highlighting but `.san` files not.
In vs code, I can use a vs code extension to make `.san` files have syntax highlighting too. 
So, in CodeSandbox, are there any ways to make `.san` files have syntax highlighting?

This is the url of my template: https://codesandbox.io/s/san-single-file-n0fex

Thanks!
Thanks a lot!
Thank you very much!
(Maybe my English is not very good. Forgive me.)

**Comment by smhutch** [comment, marked as answer]
@Lohoyo We don't currently support syntax highlighting in `.san` files. Could you share a link to the VSCode extension you use to get highlighting in VSCode please?

P.S. Your English is clear 💯

**Reply by Lohoyo** [reply]
WOW~My English is clear? I'm happy! Thank you!
This is the link to the VSCode extension I use to get highlighting in VSCode: https://marketplace.visualstudio.com/items?itemName=suyanzhi.sanny

**Reply by Lohoyo** [reply]
By the way, do you have a plan to support syntax highlighting in `.san` files, or a plan to support user-configurable syntax highlighting？If yes, when? Thanks again.

**Reply by JamesACS** [reply]
Hey @Lohoyo - It's not going to be something we'll be putting out in the near future, but certainly something we'd love to support

**Reply by ivanjeremic** [reply]
> @Lohoyo We don't currently support syntax highlighting in `.san` files. Could you share a link to the VSCode extension you use to get highlighting in VSCode please?
> 
> P.S. Your English is clear 💯

I hope that CodeSandBox will accept at least Language Extensions, this is very important and a win win situation because language makers can integrate their language in CodeSandBox and you guys have less work and less requests.
